account-kit/rn-signer/ios/implementation/NativeTEKStamperImpl.swift

```diff
@@ -41,18 +48,30 @@ enum StamperError: Error {
         }
     }
     
-    @objc public func injectCredentialBundle(bundle: NSString) async throws -> ObjCBool {
-        if let ephemeralPrivateKey = ephemeralPrivateKey {
-            let (bundlePrivateKey, bundlePublicKey) = try AuthManager.decryptBundle(encryptedBundle: bundle as String, ephemeralPrivateKey: ephemeralPrivateKey)
+     @objc public func injectCredentialBundle(bundle: NSString) async throws -> ObjCBool {
+       if let ephemeralPrivateKey = ephemeralPrivateKey {
+           
+        do {
+               let (bundlePrivateKey, bundlePublicKey) = try AuthManager.decryptBundle(
+                   encryptedBundle: bundle as String, 
+                   ephemeralPrivateKey: ephemeralPrivateKey
+               )
+               
+               apiPublicKey = bundlePublicKey
+               apiPrivateKey = bundlePrivateKey
             
-            apiPublicKey = bundlePublicKey
-            apiPrivateKey = bundlePrivateKey
-            
-            return true;
-        } else {
-            throw StamperError.notInitialized
-        }
-    }
+               return true
+           } catch let error as AuthError {
+               print("Auth error: \(error)")
```

Comment:
Do we want to print here? I actually don't think this try catch error pass up is helpful? We might want to just kill the two catch blocks. Unless there is a specific reason to do so?